docs/development.rst

```diff
@@ -194,6 +179,30 @@ Macros for Set Operations
     a ``multiunion()`` function (compute a union of many input sets at high
     speed).  This currently makes sense only for structures with integer keys.
 
+Datatypes
+=========
+
+There are two tunable values exposed on BTree and TreeSet classes.
+Their default values are found in ``_datatypes.py`` and shared across
+C and Python.
+
+
+``max_leaf_size_str``
```

Comment:
Whoops, the `_str` suffix is a mistake. That's what the internal C variable holding the python string "max_leaf_size" is called.

I'm afraid the `max_leaf_size` and `max_internal_size` names are set in stone at this point. That's what they've been called since the feature first existed.

I agree that different names make more sense, which is why I used them in `_datatypes.py` 😄 There are some functions that work with them that also follow those names (e.g., `bucket_size_for_value(value_type)`, so I'd prefer not to change `_datatypes.py`, I think that would be obfuscation. I think it would be better to live with the impedance mismatch at the one place it occurs.